### ISSUE TITLE
Game Crash when clicking Login button

### ISSUE THREAD

**Issue by idurvesh** [playgameservices/play-games-plugin-for-unity#1819]
Hi I am on latest version of this plugin.I added `            PlayGamesPlatform.Instance.Authenticate(SignInCallback, false);
`
Line to let user log in also has initialization code...

The thing is when I click a button the game crashes on device whereas if I use this line `   Social.localUser.Authenticate((bool success) => {
        // handle success or failure
    });`
The success msg do come rightly without crash though it does not give pop-up to a user on which account to choose from.

Why is it happening?

**Comment by JimmyDeemo** [comment]
Does the fix in #1319 resolve your problem?

**Comment by idurvesh** [comment]
@JimmyDeemo The problem is if you only want to use SaveGame functionality then also you have to create fake achievement in Play COnsole to get "Resources" file so Setup in unity can be completed, I think there shall be option to setup only with Applciation ID

**Comment by claywilkinson** [comment]
That's good feedback, thanks. BTW, I usually create an event (less fields to fill in :) ). You should at least looking into using the events also.  These can be sent back to the server to give you more insight into how your game is played so you can make it better.

**Comment by idurvesh** [comment]
@claywilkinson Yes will try it, currently, I am getting another error of "Google play resources not found" even though I have not changed anything

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
